fractal-puppy/Engine
# New enemy life UI

In this pr the new UI for the enemy life is added.
To test:
- Play the game and hover the enemies. You will see the new UI.
- If you changes the level of the enemy in the panel you will see different skulls in the enemy life UI. 

![Captura566](https://user-images.githubusercontent.com/7702260/67140967-89d89c00-f25f-11e9-89f6-6b48a11e73ec.PNG)
